compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -2169,6 +2170,88 @@ TR::Instruction *loadAddressConstantInSnippet(TR::CodeGenerator *cg, TR::Node *n
    return generateTrg1ImmSymInstruction(cg, TR::InstOpCode::ldrx, node, targetRegister, 0, labelSym, cursor);
    }
 
+bool shouldLoadNegatedConstant32(int32_t value)
+   {
+   int32_t negatedValue = -value;
+   if ((value >= 0 && value <= 65535) ||
+       (value >= -65535 && value < 0) ||
```

Comment:
These two lines can be merged into `if ((value >= -65535 && value <= 65535) ||`.
There are similar lines for `negatedValue` below.

Reply:
Thanks. I merged those lines and similar lines for `negatedValue`. Also removed redundant parentheses.